src/mon/MgrMap.h

```diff
@@ -179,6 +179,23 @@ class MgrMap
     return false;
   }
 
+  bool module_enabled(const std::string &module_name) const
+  {
+    return (modules.find(module_name) != modules.end());
+  }
+
+  bool any_support_module(const std::string& module) {
```

Comment:
mark this method `const`, if you please.

Reply:
"any" is singular, so should be "supports", i guess?

Reply:
Either way is probably fine.  I had been reading this as a contraction of "[do] any [daemons] support module" rather than "[if] any [daemon] supports module".  I don't even know the right grammar terminology to describe the difference though :-D